Automattic/Simplecons
# Add link icon

### Fix
- Add link icon

<img width="323" alt="Screenshot 2020-08-20 at 21 24 52" src="https://user-images.githubusercontent.com/36532468/90822098-d7181500-e32b-11ea-972c-0a2cd9ecd88a.png">

<img width="717" alt="Screenshot 2020-08-20 at 21 31 12" src="https://user-images.githubusercontent.com/36532468/90822620-a71d4180-e32c-11ea-9853-abdfcbcd2626.png">

### To test
- Verify that the icon appears as intended on a 24x24px pixel grid
- Verify that icon looks great in the set!
